.appveyor.yml

```diff
@@ -40,11 +40,13 @@ before_build:
 - cmake -H. -DCMAKE_INSTALL_PREFIX=install -Bbuild_serial -DMFEM_USE_MPI=FALSE
 
 build_script:
-- cmake --build build_parallel
-- cmake --build build_serial
-- cmake --build build_serial --target exec
+- cmake --build build_parallel --config Release -j 4
+- cmake --build build_serial --config Release -j 4
+- cmake --build build_serial --target exec --config Release -j 4
```

Comment:
Why these changes in Appveyor?

Reply:
the `-j 4` was to try to speed up the build, the `Release` is to show that by default we're building in this configuration, which has to be given for the tests on the windows platform.